doctr/travis.py

```diff
@@ -270,3 +274,27 @@ def push_docs():
         run(['git', 'push', '-q', 'doctr_remote', 'gh-pages'])
     else:
         print("The docs have not changed. Not updating")
+
+
+DOCS_REGEX = re.compile('([\w.]*)\/conf\.py')
+
+def find_build_dir():
```

Comment:
Maybe `find_sphinx_build_dir`. We might eventually want to make doctr more generic than Sphinx.